Add information about using "p" dir coming from other Dgraph clusters (#138)
dgraph-io/dgraph-docs: content/deploy/fast-data-loading/bulk-loader.md

```
Note that `snapshot at index` value must be the same within the same Alpha group and `ReadTs` must be the same value within and among all the Alpha groups.

### Using "p" directories coming from different Dgraph clusters

> In case you are using a `p` directory coming from a different Dgraph cluster (e.g. you are copying the `p` directory from your current Dgraph cluster and you want to spin up another Dgraph cluster using the same `p`) you will need to do an additional step before starting your Alpha nodes. 

Reviewer: ```suggestion
In some cases, you might want to use the `p` directory from an existing Dgraph cluster when creating a new cluster. You can do this by copying the `p` directory from your current Dgraph cluster into the new cluster, but you will need to perform an additional step before starting the Alpha nodes in the new cluster, as described below. 
```

> After starting your Zero nodes, you need to increase the timestamp of Zero by sending the following curl request to the Zero leader node:

Reviewer 2: ```suggestion
After starting your Zero nodes, you need to increase the Zero's timestamp by sending the following curl request to the Zero leader node:
```

```
curl "zero_address:port/assign?what=timestamps&num=X" #with X = high number e.g. 100000 or higher
```

> and this will print the following message:

Reviewer 2: ```suggestion
This will print the following message:
```

```
{"startId":"1","endId":"10000000","readOnly":"0"}
```

> Now you need to double-check if this timestamp has been increased correctly and this can be checked by sending a curl request to the zero `/state` endpoint:

Reviewer 2: ```suggestion
Now you need to double-check if this timestamp has been increased correctly. This can be checked by sending a curl request to the zero `/state` endpoint:
```

> Now you need to double-check if this timestamp has been increased correctly. This can be checked by sending a `curl` request to the zero `/state` endpoint:

Reviewer: ```suggestion
Now you need to confirm whether this timestamp has been increased as expected. You can do this by sending a `curl` request to the zero `/state` endpoint, as follows:
```

```
curl zero-address:port/state | jq
```

> At the very end of the response you will be looking at `"maxTxnTs"` this should be at least equal or higher than the timestamp you assigned in the previous curl request

Reviewer 2: ```suggestion
At the very end of the response, you will be looking at `"maxTxnTs"` this should be at least equal or higher than the timestamp you assigned in the previous `curl` request
```

> At the end of the response, you will be looking at `"maxTxnTs"`. This should be at least equal or higher than the timestamp you assigned in the previous `curl` request

Reviewer: ```suggestion
At the end of the response, look for the value shown for `"maxTxnTs"`. This value should be greater than or equal to the timestamp you assigned in the previous `curl` request, as shown in the following example:
```

> At the very end of the response, you will be looking at `"maxTxnTs"` this should be at least equal or higher than the timestamp you assigned in the previous `curl` request

Reviewer 2: ```suggestion
At the very end of the response, you will be looking at `"maxTxnTs"` this should be at least equal or higher than the timestamp you assigned in the previous `curl` request
```
```suggestion
At the end of the response, you will be looking at `"maxTxnTs"`. This should be at least equal or higher than the timestamp you assigned in the previous `curl` request
```

```json
  "maxLeaseId": "0",
  "maxTxnTs": "10010000",
  "maxRaftId": "0",
  "removed": [],
  "cid": "47285b0d-6223-421e-aab5-e6910620a8ed",
  "license": {
    "user": "",
    "maxNodes": "18446744073709551615",
    "expiryTs": "1619041648",
    "enabled": true
  }
```

> Only at this point you can start all your Alpha nodes and check that the Snapshot created has a `ReadTs` equal or higher that the `"maxTxnTs"`.

Reviewer: ```suggestion
Now, start your Alpha nodes and check that the Snapshot created has a `ReadTs` value that's greater than or equal to the `"maxTxnTs"` value you viewed in the previous step.
```
